### PR TITLE
improvement/upgrade-maven-version

### PR DESCRIPTION
**Pull request by carolinapc**
changed maven link

**What this PR does / why we need it**:
Upgrade maven version to 3.8.5

**Which issue(s) this PR fixes**:
Currently our builds are filing due to this old version setup. 

Fixes #

**Special notes for your reviewer**:
We'd like to later on add flexibility to set the maven version.

### CODE REVIEW

File: plugins/jib/maven.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ import execa from "execa"
 const buildLock = new AsyncLock()
 
 const spec = {
-  url: "https://archive.apache.org/dist/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.tar.gz",
+  url: "https://archive.apache.org/dist/maven/maven-3/3.8.5/binaries/apache-maven-3.8.5-bin.zip",
   sha256: "26ad91d751b3a9a53087aefa743f4e16a17741d3915b219cf74112bf87a438c5",
```
Comment (vvagaytsev) [inline plugins/jib/maven.ts:20]: Please, update the sha256 hash too:
```suggestion
  sha256: "26ad91d751b3a9a53087aefa743f4e16a17741d3915b219cf74112bf87a438c5",
  sha256: "d53e045bc5c02aad179fae2fbc565d953354880db6661a8fab31f3a718d7b62c",
```

Comment (carolinapc) [reply plugins/jib/maven.ts]: Done! Thank you!

File: plugins/jib/maven.ts [hunk 2]
```diff
@@ -16,8 +16,8 @@ import execa from "execa"
 const buildLock = new AsyncLock()
 
 const spec = {
-  url: "https://archive.apache.org/dist/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.tar.gz",
-  sha256: "26ad91d751b3a9a53087aefa743f4e16a17741d3915b219cf74112bf87a438c5",
+  url: "https://archive.apache.org/dist/maven/maven-3/3.8.5/binaries/apache-maven-3.8.5-bin.zip",
```
Comment (vvagaytsev) [inline plugins/jib/maven.ts:19]: I've just noticed that the file extension has been changed to `.zip`. These 2 lines should be placed on LL46-47 where Windows-specific maven binary is defined.

Here we should have
```
  url: "https://archive.apache.org/dist/maven/maven-3/3.8.5/binaries/apache-maven-3.8.5-bin.tar.gz",
  sha256: "88e30700f32a3f60e0d28d0f12a3525d29b7c20c72d130153df5b5d6d890c673",
```

**Review by vvagaytsev — CHANGES_REQUESTED** [review]

File: plugins/jib/maven.ts (hunk 2 above)
Comment (carolinapc) [reply plugins/jib/maven.ts]: Done, also I fixed the targetPath. Thanks

**Review by vvagaytsev — APPROVED** [review]

**Pull request merged**